client/me/help/help-contact/index.jsx

```diff
@@ -578,7 +579,7 @@ const HelpContact = React.createClass( {
 		//    requests are sent to the language specific forums (for popular languages)
 		//    we don't tell the user that support is only offered in English.
 		const showHelpLanguagePrompt =
-			( olark.locale !== i18n.getLocaleSlug() ) &&
+			( currentUserLocale !== i18n.getLocaleSlug() ) &&
```

Comment:
> It seems possible that currentUserLocale will always be the same as `i18n.getLocaleSlug()`

So it looks like the default locale is set [here](https://github.com/Automattic/wp-calypso/blob/9bcd2d242a2316851fb1cf61686cc57863b8b9b2/config/production.json#L6) to `en`. So its basically `currentUserLocale !== 'en'`